File: dotnet/C#/Calling_a_.NET_Component_from_a_COM_Component/Person.cs
interface IPerson
{
    string Name { get; set; }
    int Age { get; set; }
}

class Person : IPerson
{
    public string Name
    {
        get { return name; }
        set { name = value; }
    }

    public int Age
    {
        get { return age; }
        set { age = value; }
    }

    string name;
    int age;
}

File: XML/marshalling/Person.java
/**
* An simple person class
*/

public class Person implements java.io.Serializable
{
	/**
	* The name of the person
	*/
	
	private String name = null;
	
	
	/**
	* Creates a Person with no name
	*/
	
	public Person()
	{
		super();
	}
	
	
	/**
	* Creates a Person with the given name
	*/
	
	public Person(String name)
	{
		this.name  = name;
	}
	
	
	/**
	* @return name of the person
	*/
	
	public String getName()
	{
		return name;
	}
	
	
	/**
	* Sets the name of the person
	* @param name the name of the person
	*/
	
	public void setName(String name)
	{
		this.name = name;
	}
}

